orttraining/orttraining/core/graph/gradient_builder.cc

```diff
@@ -664,7 +664,7 @@ IMPLEMENT_GRADIENT_BUILDER(GetMulGradient) {
     } else {
       output.push_back(
           NodeDef("Identity",
-                  {IA("PreReduceGrad0")},
+                  {IA("PreReduceGrad0", OType(0))},
```

Comment:
Do we need to propagate output type for b_axes as well or one is enough?

Reply:
It's not required, but good to have.
Updated.